CHANGELOG.develop

```diff
@@ -2821,7 +2821,9 @@ Other:
   Stewart, timor, TinySong, Titov Andrey, Thomas de Beauchêne, Tomasz
   Cichocinski, tzhao11, Vincent Taing, Vlad Bokov, Vladimir Kochnev, Wieland
   Hoffmann, Witoslaw Koczewski, Xiang Ji, Yi Liu, Zane Sterling, zer09, Zhige
-  Xin, Serghei Iakovlev)
+  Xin, Serghei Iakovlev)  
+**** Documentation and website
```

Comment:
Right above this section there is `Various documentation improvements` with a list of all people that have contributed to the docs. Please add your name in that list if it does not already exist.

Reply:
done, thanks! I'll squash these in a few minutes.

Reply:
_two days after the initial squash request_, squashed.